sections/k-partners-listing.liquid

```diff
@@ -0,0 +1,467 @@
+                                                            (min-width: 990px) calc((100vw - {{ grid_space_desktop }}) * {{ image_width }} / {{ number_of_columns }}),
+                                                            (min-width: 750px) calc((100vw - {{ grid_space_tablet }}) * {{ image_width }} / {{ number_of_columns_mobile }}),
+                                                            calc((100vw - {{ grid_space_mobile }}) * {{ image_width }} / {{ number_of_columns_mobile }})
+                                                        {% endcapture %}
+                                                        {{
+                                                            object.logo
+                                                            | image_url: width: 300
+                                                            | image_tag:
+                                                            widths: '50, 75, 100, 150, 200, 300, 400, 500, 750, 1000, 1250, 1500, 1750, 2000, 2250, 2500, 2750, 3000, 3200',
+                                                            sizes: sizes
+                                                        }}
+                                                    </div>
+                                                </div>
+                                            {% else %}
+                                                <div class="tab-logo">
+                                                    <div class="placeholder-tab-img">
+                                                        {% render 'icon-setting' %}
+                                                    </div>
+                                                </div>
+                                            {% endif %}
+
+                                        {% unless services.value.size == 0 or services.value.size == blank %}
+                                            {% if section.settings.tab_subtitle %}
+                                                <h3 class="title inline-richtext h3">{{ section.settings.tab_subtitle }}</h3>
+                                            {% endif %}
+                                            <ul class="list-style-type">
+                                                {% for i in services.value %}
+                                                    <li>{{ i }}</li>
+                                                {% endfor %}
+                                            </ul>
+                                        {% endunless %}
+                                        {% if section.settings.tab_button != blank %}
+                                            <a href="{{ link | default: '#' }}" class="button button--primary button--small">{{ section.settings.tab_button }}</a>
+                                        {% endif %}
```

Comment:
I want to change the behavior on this button. What we ideally want to do is render a 3rd-party form from Hubspot into the page when the _Request a Quote_ button is pressed. Here is the URL of the form currently: `https://share.hsforms.com/1_FW9VV4vRrCxTtyIK11eagdff38`

We also need to pass a `partner_id` to the form that represents the ID of the Installer partner.

I think this will work  by embedding an `iframe` element here when the _Request a Quotation_ button is clicked:

1. When the _Request a Qutotation_ button is clicked, insert an `iframe` element in its place with the `src` of the form with partner_id. Example: `<iframe src="https://share.hsforms.com/1_FW9VV4vRrCxTtyIK11eagdff38?partner_id=15656582`></iframe>
2. store the `partner_id` as an attribute of Installer metaobject (instead of `link_to_rfq`)
3. the base link for the form (same link for all partners) should be part of the schema of the `k-partners-listing` section so it can be easily updated if we change the Hubspot form later.
4. style the iframe so that it fills the width of the partner card and expands the height.